keyboards/laptreus/keymaps/debug/keymap.c

```diff
@@ -0,0 +1,12 @@
+#include QMK_KEYBOARD_H
+#include "laptreus.h"
```

Comment:
This isn't needed.  The `QMK_KEYBOARD_H` is actually doing this. 

Specifically, when compiling the keyboard, it assigns the keyboard's h file as the `QMK_KEYBOARD_H` macro/variable, so this is redundant. 

```suggestion
```